src/layers/legacy/medCoreLegacy/parameters/medIntParameterL.h

```diff
@@ -28,12 +28,12 @@ class MEDCORELEGACY_EXPORT medIntParameterL : public medAbstractIntParameterL
     Q_OBJECT
 
 public:
-    medIntParameterL(QString name = "Unknow int parameter", QObject* parent = 0);
+    medIntParameterL(QString name = "Unknown int parameter", QObject* parent = nullptr);
     virtual ~medIntParameterL();
 
     void setRange(int min, int max);
     QSpinBox* getSpinBox();
-    QSlider* getSlider();
+    medSlider *getSlider();
```

Comment:
maybe you have two versions of medSlider:
- medSliderL
- medSlider
Here is it legacy is needed?

Reply:
We do not have a medSlicer "non legacy", since we code on "legacy" plugins. Maybe you're asking to change only the name medSlider to medSliderL? Concerning medIntParameter, it's not the same system, so it's complicated to apply medSlider to non-legacy.

Reply:
Ok for renaming to medSliderL

Reply:
It's done :)